app/src/main/java/com/android/sample/model/image/ImageViewModel.kt

```diff
@@ -50,4 +50,8 @@ class ImageViewModel @Inject constructor(private val repository: ImageRepository
   ) {
     repository.fetchActivityImagesAsBitmaps(activityId, onSuccess, onFailure)
   }
```

Comment:
Added a new method deleteProfilePicture to the ImageViewModel to delegate the deletion to the repository.